docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeBinning.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.NormalizingTransformer;
```

Comment:
do we need this?

Reply:
Yes, otherwise I need to write Microsoft.ML.Transforms.NormalizingTransformer.BinNormalizerModelParameters which takes too much space

---
In reply to: [274057369](https://github.com/dotnet/machinelearning/pull/3244#discussion_r274057369) [](ancestors = 274057369)